README.md

```diff
@@ -171,6 +170,40 @@ var userInfo = new RaygunIdentifierMessage("12345")
 
 Log.ForContext("CustomUserInfoProperty", userInfo, true).Error(new Exception("random error"), "other information");
 ```
+
+### onBeforeSend
+`type: Action<OnBeforeSendParameters>`
+
+`default: null`
+
+The field allows users to manipulate the crash report payload to be sent to Raygun.
```

Comment:
"This action allows you to manipulate the crash report payloads that get sent to Raygun."